src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -244,7 +263,10 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging {
         _ <- saveWorkspace(dataAccess, updatedEntitiesAndWorkspace)
         _ <- saveEntities(dataAccess, workspace, updatedEntitiesAndWorkspace)
         _ <- saveErrors(updatedEntitiesAndWorkspace.collect { case Right(errors) => errors }, dataAccess)
-      } yield Unit
+      } yield Unit)  map (_ => Unit)
+      //map to Unit? apparently so, otherwise scala thinks the return type of this function is:
+      //DBIOAction[Unit.type, NoStream, Read with Read with Read with Read with Write with Read with Write with Read with Write with Write]
+      //...not what we mean.
```

Comment:
just to be extra clear here: this insane generated type is not compatible with `ReadWriteAction[Unit]`, either. It's its own monster.